manifests/database/postgresql.pp

```diff
@@ -68,10 +68,12 @@
     }
   }
 
+  $db_port = pick($database_port, '5432')
```

Comment:
instead of the pick, I think it makes sense to set $database_port to 5432 as default value?

Reply:
Yeah, the previously mentioned line 48 would be deleted, and the test adjusted like I pushed just now. Is it what you have in mind?

(I just figured out how the suggestion in github worked, I could have make a suggestion directly here, sorry!)